<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConstellationSelectionView</class>
 <widget class="QWidget" name="ConstellationSelectionView">
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QCheckBox" name="beidouCheckBox">
     <property name="text">
      <string>BEIDOU</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="navicCheckBox">
     <property name="text">
      <string>NAVIC</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="sbasCheckBox">
     <property name="text">
      <string>SBAS</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="gpsCheckBox">
     <property name="text">
      <string>GPS</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="qzssCheckBox">
     <property name="text">
      <string>QZSS</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="glonassCheckBox">
     <property name="text">
      <string>GLONASS</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="galileoCheckBox">
     <property name="text">
      <string>GALILEO</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="updateConstellationsButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
